components/SettingsControls/src/Helpers/IsNullOrEmptyStateTrigger.cs

```diff
@@ -0,0 +1,141 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Specialized;
+
+#if WINAPPSDK
+using CommunityToolkit.WinUI.Helpers;
+#else
+using Microsoft.Toolkit.Uwp.Helpers;
+#endif
+
+
+namespace CommunityToolkit.Labs.WinUI;
+
+/// <summary>
+/// Enables a state if an Object is <c>null</c> or a String/IEnumerable is empty
+/// </summary>
+public class IsNullOrEmptyStateTrigger : StateTriggerBase
```

Comment:
We should consider making this internal, to avoid conflicting with helpers in other components down the road (including the inbox `IsNullOrEmptyStateTrigger` that'll get ported soon)

Reply:
Can it be referenced from XAML still if `internal`? I thoughts that why we weren't doing it. I mean ideally, we just get these ported over in the next week or two and then just start referencing preview packages for them in labs components?

Reply:
Yeah the idea is to remove these custom triggers until we have ported them over and made the fixes there.